lib/bootstrap/bundler.rb

```diff
@@ -115,6 +111,7 @@ def invoke!(options = {})
       require "bundler"
       require "bundler/cli"
 
+      require "fileutils"
```

Comment:
all the requires are used in one specific place - so it makes sense to have them inline for a bootstrap script ...
e.g. the `require "fileutils"` will not be loaded during LS boot (only loads with `bundle install`)